### DISCUSSION TITLE
Wrapped %sveltekit.body% ?

### DISCUSSION THREAD

**Discussion by MentalGear** [sveltejs/kit#7569] [Q&A] [answered]
In the examples, in `app.html` `%sveltekit.body%` is wrapped inside a `<div>`. Is this relevant, or can we delete the outer div?

**Comment by geoffrich** [comment, marked as answer]
The reason for this is in the [docs on project structure](https://kit.svelte.dev/docs/project-structure#project-files-src)

> %sveltekit.body% — the markup for a rendered page. Typically this lives inside a `<div>` or other element, rather than directly inside <body>, to prevent bugs caused by browser extensions injecting elements that are then destroyed by the hydration process

**Reply by MentalGear** [reply]
Ok, thx, good to know!
